engine/admin/logging/overview.md

```diff
@@ -89,6 +89,26 @@ json-file
 {% endraw %}
 ```
 
+## Configure the delivery mode of log messages from container to log driver
+
+Docker provides two modes for delivering messages from the container to the log driver since Docker 17.04:
+1. (default) direct, blocking delivery from container to driver
+2. non-blocking delivery that stores log messages in an intermediate per-container ring buffer
+
+The non-blocking mode of message delivery is useful for ensuring that applications will not block when the logging sub-system is unable to keep up with message throughput.
```

Comment:
Maybe "for preventing applications from blocking when..."

Reply:
I'd replace all of line 98 with: 

> This message delivery mode will prevent applications from blocking due to logging back pressure. Applications - especially network applications - will likely fail in unexpected ways when STDERR and STDOUT IO blocks.

Reply:
Thank you for your feedback Misty and Jeff.

I adjusted the docs to use a more active voice and to try and match the suggested style.  I referred to test.md to try and get the warning admonition correct.

While preparing these updates, I found documentation of these features by cpuguy83 (author) in PR https://github.com/docker/docker.github.io/pull/1482

It looks like those docs were merged to the `vnext-engine` branch in 144f9ab and then were removed somewhere along the way.

Misty, if you would prefer to use/restore those docs, it's fine with me.  I just want the options documented :)